flow/project.py

```diff
@@ -3385,10 +3862,19 @@ def _register_groups(self):
         # Gather all groups from class and parent classes.
         for cls in type(self).__mro__:
             group_entries.extend(getattr(cls, '_GROUPS', []))
-
+        flow_aggregate = dict()
+        flow_select = dict()
+        for op_name, op in self._operations.items():
+            try:
+                flow_aggregate[op_name] = self._operation_functions[op_name]._flow_aggregate
+                flow_select[op_name] = self._operation_functions[op_name]._flow_select
+            except KeyError:
+                flow_aggregate[op_name] = op.cmd._flow_aggregate
+                flow_select[op_name] = op.cmd._flow_select
```

Comment:
This code likely isn't necessary. This can be done in the loop below that uses `(op_name, op)` and the `func` variable.